aiida/orm/implementation/querybuilder.py

```diff
@@ -9,6 +9,12 @@
 ###########################################################################
 """Backend query implementation classes"""
```

Comment:
Maybe note here that the general QueryBuilder here is based on the assumption that all queries are performed with SQLAlchemy, no matter the actual backend? I.e., just copy some stuff in here from your commit messages.